clients/tabby-agent/src/AgentConfig.ts

```diff
@@ -12,6 +12,8 @@ export type AgentConfig = {
     requestTimeout: number;
   };
   proxy: {
+    enabled: boolean;
```

Comment:
I prefer to not add `proxy.enabled`. You can set the `proxy.url` to empty to disable the proxy.

Reply:
Updated.